bcgov/HOUS-permit-portal
# Feature/bphh 1174/reinvite

## Description

Add a third array of users in the user inviter, "reinvited"
this allows unconfirmed or discarded uses to be invited again.

add another user management menu option that opens invite with the confirmed or archived users' info pre-populated.

remove unused "resend" endpoint and translations.

bonus: fix jurisdiction links

https://hous-bssb.atlassian.net/browse/BPHH-1174
https://hous-bssb.atlassian.net/browse/BPHH-736

Testing steps:

log in as review manager, go to user manage table
archive a user
click show archived users
click manage user > re-invite user
see the pre-populated data
invite the existing user
new invite should be sent, and the row should show "re-invited" success status
